AppCenter/AppCenter/MSACAppCenter.h

```diff
@@ -151,38 +121,33 @@
 /**
  * Check whether the application delegate forwarder is enabled or not.
  *
- * @return YES if enabled, NO otherwise.
- *
  * @discussion The application delegate forwarder forwards messages that target your application delegate methods via swizzling to the SDK.
  * It simplifies the SDK integration but may not be suitable to any situations. For
  * instance it should be disabled if you or one of your third party SDK is doing message forwarding on the application delegate. Message
  * forwarding usually implies the implementation of @see NSObject#forwardingTargetForSelector: or @see NSObject#forwardInvocation: methods.
  * To disable the application delegate forwarder just add the `AppCenterAppDelegateForwarderEnabled` tag to your Info .plist file and set it
  * to `0`. Then you will have to forward any application delegate needed by the SDK manually.
  */
-+ (BOOL)isAppDelegateForwarderEnabled;
+@property(class, readonly, nonatomic) BOOL isAppDelegateForwarderEnabled;
```

Comment:
ObjC properties should not be prefixed by "is". In ObjC only the getter should be prefixed by "is". Example @property (assign, getter=isEditable) BOOL editable; cf Apple doc. The good new is it won't break ObjC customers :).